bedrock/mozorg/helpers/misc.py

```diff
@@ -517,3 +518,8 @@ def slugify(text):
     trailing whitespace.
     """
     return django_slugify(text)
+
+
+@jingo.register.filter
+def bleach_tags(text):
+    return bleach.clean(text, tags=[], strip=True).replace('&amp;', '&')
```

Comment:
I thought about that, and then assumed that the explicit `|safe` was a good thing, but maybe doing that together with renaming to `bleach_safe`?

Yes, bleach.clean() converts ampersands and I didn't see a way to tell it not to, so I just convert them back.